src/java/org/apache/cassandra/db/compaction/CompactionManager.java

```diff
@@ -2537,4 +2608,104 @@ public interface CompactionPauser extends AutoCloseable
     {
         public void close();
     }
+
+    public static class OperationInfo
+    {
+        /** The unique id for this operation */
+        public final TimeUUID operationId = TimeUUID.Generator.nextTimeUUID();
+        /** The type of operation being performed */
+        public final OperationType operationType;
+        /** The keyspace names for which this operation is being performed */
+        public final Set<String> keyspaceNames = new ConcurrentSkipListSet<>();
+        /** The keyspace and table names for which this operation is being performed */
+        public final Map<CFInfo, AllSSTableOpStatus> processedCfs = new ConcurrentHashMap<>();
+        /** The number of sstables that have been completed for each keyspace */
+        public final Map<String, AtomicInteger> processedSStables = new ConcurrentHashMap<>();
```

Comment:
`numberOfSSTablesPerKeyspace`, `sstableCountPerKeyspace`, `processedSSTablesPerKeyspace` ... you got the idea

Reply:
Fixed.